docs/_sidebar.md

```diff
@@ -0,0 +1,14 @@
+* [Introduction](/)
+* [Getting started: the walkthrough](walkthrough.md)
```

Comment:
Put part 1 and 2 as sub pages of this?

Reply:
Confused - the walkthrough is in a single file. I am not sure it can be neatly divided into two parts.